response_operations_ui/controllers/message_controllers.py

```diff
@@ -69,6 +69,20 @@ def send_message(message_json):
         raise ApiError(ex.response)
 
 
+def remove_unread_label(message_id):
+    url = f'{current_app.config["BACKSTAGE_API_URL"]}/v1/secure-message/update-label/{message_id}'
+    json = dumps({"label": "UNREAD", "action": "remove"})
+
+    logger.debug("Removing message unread label", message_id=message_id)
+    response = requests.put(url, headers={"Authorization": _get_jwt(), "Content-Type": "application/json"}, data=json)
```

Comment:
Will this work against the current implementation? I was under the impression that it only accepted GET and POST